src/components/files/ListingItem.vue

```diff
@@ -172,9 +167,12 @@ export default {
       }
     },
     open: function() {
-      this.$store.commit('pushFolder', { id: this.id, name: this.name });
-      this.$store.commit('setReload', true);
-      
+      if (api.isDocumentEdit(this.type)) {
```

Comment:
(like the other comment) you can use the function from the utils "checkDocumentEdit".

Reply:
Also, I saw that it appears in the Listing.vue file as well. Maybe instead of duplicate this checks you can create one function.